Tests/Integration/CloudToDeviceMessageTests.cs

```diff
@@ -1076,7 +1014,7 @@ public async Task When_Takes_Too_Long_Getting_FcntDown_Should_Abandon_Message()
             // 3. Device FcntDown did change
             var devices = loRaDeviceRegistry.InternalGetCachedDevicesForDevAddr(simulatedDevice.DevAddr);
             Assert.True(devices.TryGetValue(simulatedDevice.DevEUI, out var loRaDevice));
-            Assert.Equal(InitialDeviceFcntDown + 1, loRaDevice.FCntDown);
+            Assert.Equal(InitialDeviceFcntDown + Constants.MaxFcntUnsavedDelta, loRaDevice.FCntDown);
```

Comment:
I don't get how we can end up with + 10 here?

Reply:
It's not something that I changed, it came from the existing test. It comes from `SingleGatewayFrameCounterUpdateStrategy.Initialize`.

Reply:
did you debug this? How does it get bumped up? Let me know if you want to step through it.

Reply:
As per our discussion, will resolve this now